examples/gemma/gemma.yml

```diff
@@ -0,0 +1,66 @@
+# use google/gemma-7b if you have access
+base_model: mhenrichsen/gemma-7b 
+model_type: AutoModelForCausalLM
+tokenizer_type: AutoTokenizer
+ 
+load_in_8bit: false
+load_in_4bit: true
+strict: false
+ 
+datasets:
```

Comment:
change to something from hf. A lot of the examples use mhenrichsen/alpaca_2k_test